reference/shipping.v3.yml

```diff
@@ -397,6 +465,24 @@ components:
           readOnly: true
           format: date-time
           example: 2022-09-21T14:15:00+00:00
+    checkoutSettings:
+      title: checkoutSettings
+      description: Data about the checkout settings object.
```

Comment:
This seems like a redundant comment.
Is there some value we can add here to be more descriptive of what one might find in these settings?

Reply:
Perhaps adding info that it's shipping specific checkout settings?

Reply:
Thanks gents, Is `Checkout settings related to shipping.` descriptive enough?